lib/px/pxnginx.lua

```diff
@@ -26,18 +26,20 @@ local M = {}
 M.configLoaded = false
 function M.application(file_name)
     local config_file = ((file_name == nil or file_name == '') and "px.pxconfig" or "px.pxconfig-" .. file_name)
+    local config_builder = require("px.utils.config_builder");
 
-    local px_config = require(config_file)
+    local px_config_file = require(config_file)
```

Comment:
following comment above, this can be dropped as the config file will be included outside nginx ad decouple it